### PR TITLE
docs: Kubernetes installation

### LINKED ISSUE #1801: Update Lagoon installation documentation
Per a rocketchat convo, it appears our documentation on installing Lagoon on OpenShift is out of date (and probably needs to be updated for K8s?). Let's get this updated. 

Current page: https://lagoon.readthedocs.io/en/latest/administering_lagoon/install/
and requirements - i assume these may change as well: https://lagoon.readthedocs.io/en/latest/administering_lagoon/openshift_requirements/

### PR DESCRIPTION
**Pull request by xantrix**
# Checklist
- [x] Affected Issues have been mentioned in the Closing issues section
- [x] Documentation has been written/updated
- [ ] PR title is ready for changelog and subsystem label(s) applied

Add/Update documentation on installing Lagoon on Kubernetes.
Still WIP, feedback required.

# Closing issues
Should close #1801

### CODE REVIEW

**Comment by tobybellwood** [comment]
Thanks @xantrix we'll have a look at this and get you some comments 😁

**Comment by xantrix** [comment]
Thanks a lot, @tobybellwood :) I'm still checking the overall process of deploying Lagoon on a Kubernetes cluster (is still ongoing right? I bumped into #1863 ). 
Happy to complete the docs as soon I'll have feedback that I'm on the right path.

File: docs/administering_lagoon/install_k8s.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
+# Install Lagoon on Kubernetes
```
Comment (tobybellwood) [inline docs/administering_lagoon/install_k8s.md]: #
```suggestion
# Install local Kubernetes cluster for Lagoon
```

File: docs/administering_lagoon/install_k8s.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+Let's see how to install Lagoon on a local lightweight k8s cluster using
```
Comment (tobybellwood) [inline docs/administering_lagoon/install_k8s.md]:
```suggestion
Let's see how to install a local lightweight k8s cluster using
```

File: docs/administering_lagoon/install_k8s.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+    mariadbprovider	 dbaas-operator
+    nginx          	 kube-system
+    ```
+
+## Deploy Lagoon on Kubernetes
+1. TODO
+
+## Configure Installed Lagoon
+
+We have a fully running Lagoon. Now it's time to configure the first project inside of it. Follow the examples in [GraphQL API](graphql_api.md).
```
Comment (tobybellwood) [inline docs/administering_lagoon/install_k8s.md]:
```suggestion
We have a fully running Kubernetes cluster. Now it's time to configure the first project inside of it. Follow the examples in [GraphQL API](graphql_api.md).
```

**Comment by tobybellwood** [comment]
This is a great start!  It's probably important to note at this stage that this process only sets up a local Kubernetes cluster for use with Lagoon testing - it doesn't install Lagoon itself (that still has to be installed on Minishift to test).

**Comment by xantrix** [comment]
> This is a great start! It's probably important to note at this stage that this process only sets up a local Kubernetes cluster for use with Lagoon testing - it doesn't install Lagoon itself (that still has to be installed on Minishift to test).

Thanks @tobybellwood :bow: just merged your suggested changes. 
Thank you for confirming, trying to deploy Lagoon on the local cluster I noticed that there are still differences from OpenShift setup :+1:   

Let me know if there is anything else I have to add here.

**Review by tobybellwood — APPROVED** [review]
This is a good first step.  Community feedback indicates that these steps are already the best documentation on local k8s.  We can iterate from this!

**Pull request merged**